code/ui/components/src/components/tabs/tabs.tsx

```diff
@@ -119,6 +119,8 @@ export interface TabsProps {
   }>[];
   id?: string;
   tools?: ReactNode;
+  showToolsWhenEmpty?: boolean;
+  customEmptyState?: ReactNode;
```

Comment:
nit: not a fan of `custom` in props, I feel like it's implicit that a prop customizes something, so it can just be `emptyState`.